File: Java-lecture-220820/src/package0917/File02_Loop.java
package package0917;

public class File02_Loop {
	public static void main(String[] args) {
		// 중첩 반복문
		//  - 반복문 내에 또 다른 반복문을 작성한 형태
		
		// 한 반에 30명인 학생의 번호를 출력
		for( int n = 1; n <= 30; n++) {
			System.out.println(n + "번 학생");
		}
		//-------------------------------------------
		// 3개의 반의 학생들의 번호를 출력
		// 이때 각 반에 30명의 학생이 있다.
		
		// 1반부터 3반까지
		for(int class_ = 1; class_ <= 3; class_++) {
			
			System.out.println(class_ + "반 ");
			
			// 각 반의 1번부터 30번까지
			for (int n = 1; n <= 30; n++) {
				System.out.println("\t" + n + "번 학생");
			}
		}
	
		
		
	}
}

File: lecture-c-20220514/lecture-c-20220514/06-20220529/File02_Loop.c
#include <stdio.h>

/*
	File Name		: File02_Loop.c
	Data					: 2022.05.29
*/

int main()
{
	// 반복문(Loop Statement)
	//	 코드를 반복 수행하는 제어문

	// "Hello World!" 를 다섯 번 출력
	printf("Hello World!\n");
	printf("Hello World!\n");
	printf("Hello World!\n");
	printf("Hello World!\n");
	printf("Hello World!\n");
	printf("---------------------------\n");

	// while 문
	//	 : "~ 하는 동안" 의미
	//	 - 특정 조건을 만족하는 동안 반복 수행

	// 반복문을 이용하여
	// "Hello World!" 를 다섯 번 출력

	// 카운팅
	//  - 카운팅 정보를 저장하기 위한 변수
	//  - 카운팅 검사하는 조건식
	//  - 카운팅 연산식

	// 출력횟수를 저장하기 위한 변수 = 카운팅 횟수
	int currentCount = 0;
	int printCount = 5;
	//  조건 : 출력 횟수가 5보다 작으면 반복 수행
	while (currentCount < printCount)
	{
		printf("%d : Hello World!\n", currentCount+1);
		
		// 출력했으니 카운팅
		currentCount++;
	}

	//--------------------------------------------
	// 1<= num <= 10 의 정수를 모두 출력
	int num = 1;

	while (num <= 10)
	{
		printf("num is %d.\n", num);
		num++;
	}


	return 0;
}